README.md

```diff
@@ -76,6 +77,14 @@ The artwork used on the system view can be customized with your own images.
 
 > If you create a set of images that you would like to share with the community please let me know about it [here](https://retropie.org.uk/forum/topic/33010/theme-art-book-next)
 
+### Splash Screen Background
```

Comment:
Few changes to make:
- Please to `Splash Screen` instead of `splash screen background`
- update `foledr` to `folder`